Doc/library/curses.rst

```diff
@@ -562,13 +572,13 @@ The module :mod:`curses` defines the following functions:
 .. function:: tigetstr(capname)
 
    Return the value of the string capability corresponding to the terminfo
-   capability name *capname*.  ``None`` is returned if *capname* is not a string
+   capability name *capname* as a byte string.  ``None`` is returned if *capname* is not a string
    capability, or is canceled or absent from the terminal description.
 
 
 .. function:: tparm(str[, ...])
 
-   Instantiate the string *str* with the supplied parameters, where *str* should
+   Instantiate the byte string *str* with the supplied parameters, where *str* should
```

Comment:
Hmm.  I guess you are right.  The termcap docs are a bit ambiguous in how they use the term 'capability'  Since 'str' is in the tigetstr name, if you are sticking with bytes string it makes sense to keep it.